### PR TITLE
Conformance results for v1.7/ics

### PR DESCRIPTION
**Pull request by kitch**
vendor: IBM
name: IBM Cloud Container Service
version: v1.7.4
website_url: https://www.ibm.com/cloud/container-service
documentation_url: https://console.bluemix.net/docs/containers/container_index.html
product_logo_url: https://raw.githubusercontent.com/ibm-containers/logo/master/iccs.png

### CODE REVIEW

**Comment by kbarnard10** [comment]
+1

**Comment by bradtopol** [comment]
Looks great @kitch Thanks!!!

File: v1.7/ics/PRODUCT.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+vendor: IBM
+name: IBM Container Service
```
Comment (rtheis) [inline v1.7/ics/PRODUCT.yaml]: I believe this should now be `IBM Cloud Container Service`.  Given this, should we rename the directory to iccs?

File: v1.7/ics/PRODUCT.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+version: v1.7.4
+website_url: https://www.ibm.com/Bluemix/Containers
```
Comment (rtheis) [inline v1.7/ics/PRODUCT.yaml]: https://www.ibm.com/cloud/container-service

File: v1.7/ics/README.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+To reproduce:
+
+```
+% bx cs cluster-create --name conformance
+% $(bx cs cluster-config conformance | grep export)
+% curl -L https://raw.githubusercontent.com/cncf/k8s-conformance/master/sonobuoy-conformance-1.7.yaml | kubectl apply -f -
```
Comment (rtheis) [inline v1.7/ics/README.md]: `-1.7` has been dropped from the file name.

File: v1.7/ics/PRODUCT.yaml [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+documentation_url: https://console.bluemix.net/docs/containers/container_index.html
+product_logo_url: http://ibm.biz/containerslogo
```
Comment (rtheis) [inline v1.7/ics/PRODUCT.yaml]: Hopefully it is just me, but http://ibm.biz/containerslogo is coming back with a blank screen.

File: v1.7/ics/PRODUCT.yaml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+vendor: IBM
+name: IBM Container Service
+version: v1.7.4
```
Comment (rtheis) [inline v1.7/ics/PRODUCT.yaml]: We aren't using `v` in our version numbers.

File: v1.7/ics/README.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+% kubectl logs -f -n sonobuoy sonobuoy
+
+# wait for "no-exit was specified, sonobuoy is now blocking"
+
+% kubectl cp sonobuoy/sonobuoy:/tmp/sonobuoy ./results
```
Comment (rtheis) [inline v1.7/ics/README.md]: I believe you need to drop the `/` in `/tmp/sonobuoy`.  At least, I did when I ran.

Comment (kitch) [reply v1.7/ics/README.md]: Yeah, I need to rerun, i think the paths changed in a recent conformance update.

File: v1.7/ics/README.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+To reproduce:
+
+You'll first need to get access to the IBM Cloud Container Service and acquire the appropriate CLIs.
+1) Sign up at https://bluemix.net
+2) Follow this guide to set up your CLI environment: https://console.bluemix.net/docs/containers/cs_cli_install.html#cs_cli_install_steps
+3) Follow the instructions below to create a cluster and run conformance tests.
+
+```
+% bx cs cluster-create --name conformance
```
Comment (rtheis) [inline v1.7/ics/README.md]: We should probably add `--kube-version 1.7` now.

**Pull request merged**